modules/openapi-generator/src/test/resources/3_0/python/petstore-with-fake-endpoints-models-for-testing-with-http-signature.yaml

```diff
@@ -2422,3 +2422,21 @@ components:
                 items:
                   type: string
                   nullable: true
+    Mole:
+      type: object
+      required:
+        - blind
+        - smell
+      properties:
+        blind:
+          type: boolean
+          readOnly: true
+        smell:
+          type: string
+          readOnly: false
+        touch:
+          type: boolean
+          readOnly: true
+        taste:
+          type: string
+          readOnly: false
```

Comment:
This spec example looks great! Thank you for adding it?
What do you think about also including a parameter that omits defining readOnly?

Reply:
yes I will do